tests/containers/openshift_setup.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Set-up OpenShift cluster on the host
+# There are some requirements that need to be pre-setup on the host:
+#  - user bernhard must be existing with default testapi password
+#  - user bernhard must have "ALL=(ALL) NOPASSWD: ALL" in visudo to avoid prompt input
+#  - HDD must be at least 30 or 35GB in size and min RAM 15GB, and cpu=host
+#  - pull secret must be available in some internal repository defined in OPENSHIFT_CONFIG_REPO
+#  - the repository must contain 3 things: crc-linux-amd64.tar.xz oc.rpm and ps.json
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+    my $user = $testapi::username;
+
+    # Make sure user has access to tty group
+    my $serial_group = script_output("stat -c %G /dev/$testapi::serialdev");
+    assert_script_run("chown $user /dev/$testapi::serialdev && gpasswd -a $user $serial_group");
+
+    # Get packages from repository
+    my $config_repo = get_required_var('OPENSHIFT_CONFIG_REPO');
+    assert_script_run("git clone -q --depth 1 $config_repo /root/openshift_config");
+    assert_script_run('cd /root/openshift_config');
+
+    # CRC client installation
+    assert_script_run('tar xvf crc-linux-amd64.tar.xz');
+    assert_script_run('cp crc-linux-*-amd64/crc /usr/sbin && chmod +x /usr/sbin/crc');
+    my $crc_version = script_output('crc version');
+    record_info('crc', $crc_version);
+    record_soft_failure('poo#0000000 There is a new version of CRC, please update the RPM') if ($crc_version =~ /A new version/);
```

Comment:
```suggestion
    my $msg = 'There is a new version of CRC, please update the RPM';
    record_soft_failure($msg) if ($crc_version =~ /A new version/);
```

This also makes it happy :wink:

Reply:
I thought CI would complain that there is no ticket referenced..

Reply:
changed, let's see...

Reply:
meeeehh
```
tests/containers/openshift_setup.pm:    record_soft_failure($msg) if ($crc_version =~ /A new version/);
make: *** [Makefile:147: test-soft_failure-no-reference] Error 1
```